syntax="proto2";

package aasdk.proto.ids;

message SensorChannelMessage
{
  enum Enum
  {
    NONE = 0x0000;
    SENSOR_START_REQUEST = 0x8001;
    SENSOR_START_RESPONSE = 0x8002;
    SENSOR_EVENT_INDICATION = 0x8003;
  }
}

option optimize_for = SPEED;
